ios/brave-ios/Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -109,20 +111,27 @@ struct SubmitReportView: View {
   @MainActor func createAndSubmitReport() async {
     let domain = Domain.getOrCreate(forUrl: url, persistent: !isPrivateBrowsing)
 
-    let report = WebcompatReporter.Report(
-      cleanedURL: url,
+    await WebcompatReporterUploader.send(
+      braveVersion: String(
+        format: "%@ (%@)",
+        AppInfo.appVersion,
+        AppInfo.buildNumber
+      ),
```

Comment:
channel is missing?

Reply:
ah, I see it's filled later in `WebcompatReporter.swift`. Not sure why those are separated.. maybe everything can be filled in one place?

Reply:
yeah, I removed `WebcompatReporter.swift`, so now all things are in the same place